src/pages/Bridge/BurnForm/index.tsx

```diff
@@ -112,8 +113,14 @@ const BurnForm = (): JSX.Element | null => {
     (async () => {
       try {
         setStatus(STATUSES.PENDING);
-        const theBurnRate = await window.bridge.redeem.getBurnExchangeRate(COLLATERAL_TOKEN);
+        const [theBurnRate, maxBurnableTokens] = await Promise.all([
```

Comment:
@nud3l 
We could rename `maxBurnableTokens` `theBurnableTokens` for consistency.

Reply:
That'd be nice, thanks!